### DISCUSSION TITLE
BUG(VSC 1.75): Copilot cannot accept words beyond current line

### DISCUSSION THREAD

**Discussion by corneliusroemer** [community/community#46873] [Copilot Conversations]
### Select Topic Area

Bug

### Body

In VSC release 1.75 a new command was introduced: `editor.action.inlineSuggest.acceptNextWord`

This is nice, but it only works until the end of the line, while most suggestions are multiple lines for me. So then I'm stuck and have to accept all or nothing.

This seems like a bug. Maybe in interaction with using the VSCodeVim extension I'm using.

See screencast (I've changed default keybindings for accept next word to be easier to type). New words are only accepted till line ending:

![2023-02-09 16 46 31](https://user-images.githubusercontent.com/25161793/217863524-302b5bad-6d67-4a2c-9251-286b4adf88c5.gif)

**Comment by iamcjwilliams** [comment]
@corneliusroemer Thank you for your feedback, Our copilot team has been notified
